src/test/java/school/hei/haapi/integration/conf/CallerData.java

```diff
@@ -0,0 +1,14 @@
+package school.hei.haapi.integration.conf;
+
+public class CallerData {
+  public static final String STUDENT1_ID = "student1_id";
+  public static final String STUDENT2_ID = "student2_id";
+  public static final String TEACHER1_ID = "teacher1_id";
+  public static final String TEACHER2_ID = "teacher2_id";
+  // This token expires 2021/08/25 16:00
+  public static final String STUDENT1_TOKEN =
+      "eyJraWQiOiJPam5ZXC9YVW9FVzlNVkwzbWs2bTR1a2xqMHFOdUE0WXdRRlpTbzd2MGNUOD0iLCJhbGciOiJSUzI1NiJ9.eyJhdF9oYXNoIjoiaTg4YTNHVWhaWGsxZmlGV2JaTWZPUSIsInN1YiI6IjZmYWY4ZjllLWQ2ZDQtNDE0Zi05MjE3LTMzZWEyZGRiNjczNCIsImVtYWlsX3ZlcmlmaWVkIjp0cnVlLCJpc3MiOiJodHRwczpcL1wvY29nbml0by1pZHAudXMtZWFzdC0yLmFtYXpvbmF3cy5jb21cL3VzLWVhc3QtMl9nS2RZM1JoMFgiLCJwaG9uZV9udW1iZXJfdmVyaWZpZWQiOnRydWUsImNvZ25pdG86dXNlcm5hbWUiOiJyeWFuIiwiYXVkIjoiMzd0aWhnNGZmaDU3ZGp2NDNqcDNoNDd2bWYiLCJldmVudF9pZCI6IjE2ZjhmNjhjLTA2YzctNDVjNS1hYjZjLTUyYjU5ZTVjZmFjNiIsInRva2VuX3VzZSI6ImlkIiwiYXV0aF90aW1lIjoxNjI5ODEzMTEzLCJwaG9uZV9udW1iZXIiOiIrMjYxMzQxMjM0NTY3IiwiZXhwIjoxNjI5ODk5NTEzLCJpYXQiOjE2Mjk4MTMxMTMsImp0aSI6Ijc3ZGE2ZTQyLTRmNzQtNGMyMS05ZTU0LTNkZWU0ZTFlNWJiZCIsImVtYWlsIjoicnlhbkBoZWkuc2Nob29sIn0.ehnhU47AtNY1-PTckT8nl9FWkOkyqbb2HjiciWX2HZih8NCgGoEjIarOCJWZOJ_6EiTOEvh1gWG0umeOK6z8NJD2vEPZ_fO-SkptdOsrExgtut5bzhvDdcJ4TY4MfhPqkbrFmRKQr76joE_IniudErWBglU8rv6ILTS95WufXygXyBLZ2f-Slbm-ZcaCaJ22EPzGifr62rfyfa2QMB2oq-dg0L9V7bcP7Qdaj4_-0s_-uaG3cTm5_v0WzGuTtp2EH0LB2a5YD4FCWaEYLeQl0_9uF20aYnDoOc3lE7rFbmSWKUFcZYPrQUDNutHvAiUJPC23olBHrLgWpGtOA83afg";
```

Comment:
So that it is clear once and for all: never EVER EVER EVER put authentication token in public read like this!!!!!

Put them in env variables.

We will eventually use [localstack](https://github.com/localstack/localstack) to emulate Cognito locally, but that's in a distant future. For now, just use direct call to  online Cognito as you already does, but separate its tests from the authorization test.

Reply:
the "its" in "its tests" refer to what ? 
I don't really understand what should I separate from the authorization test as our tests  are _all_ based on authorization.

Reply:
**Test class 1 : Authentication**, with direct call to Cognito. You don't test the API. You don't need Postgres. You just test the CognitoComponent class.

**Test class 2 : Authorization**, with mock Cognito. You suppose that CoginoComponent is working properly and now you the API and the DB.